server/src/main/java/io/druid/server/metrics/MonitorsConfig.java

```diff
@@ -52,6 +52,22 @@ public String toString()
            '}';
   }
 
+
+  public static Map<String, String[]> mapOfDimensionAndTaskID(
+      final String dimension,
+      final String taskId
+  )
+  {
+    final Map<String, String[]> dimensionMap = new HashMap<>();
```

Comment:
you could use an ImmutableMap.builder to avoid creating an entire hashmap just for this, but either way is fine